app/(blog)/blog/page.tsx

```
@@ -68,10 +71,15 @@ function BlogContent() {
         📚 {tagConverter(tag)} ({blogList?.totalElement})
       </S.TapTitle>
       <S.PostContainer>
-        {blogList?.data.map((post) => (
-          <PostBox key={post._id} post={post} />
-        ))}
+        {blogList?.data.length || 0 > 0 ? blogList?.data.map((post) => <PostBox key={post._id} post={post} />) : <div>no data</div>}
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)